radio/src/CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ option(TRACE_LUA_INTERNALS "Turn on traces for Lua internals" OFF)
 option(FRSKY_STICKS "Reverse sticks for FrSky sticks" OFF)
 option(NANO "Use nano newlib and binalloc")
 option(NIGHTLY_BUILD_WARNING "Warn this is a nightly build" OFF)
+option(FLEX_R9M "Add R9M options for non certified firmwwares" OFF)
```

Comment:
Any objection to changing this to `MODULE_R9M_FLEX` so it groups nicely with `MODULE_R9M_FULLSIZE`?

Reply:
Hum, full-size refers to an actual module type while flex to a firmware, so it would more something like FIRMWARE_R9M_FLEX if anything imho

Reply:
True.  It's firmware of the module, don't want to confuse that with OTX firmware though (the other FIRMWARE_* options)...  maybe one of: `MODULE_[FW_]R9M_[FW_]FLEX[FW]`?  :)   I think it relates to "MODULE" at least, the most out of any other current prefixes.  When using a GUI (or any alpha-sorted list view) it's nice to have groups/organization.

Reply:
I would also vote for MODULE_R9M_FLEX or MODULE_R9M_FLEX_FW

Reply:
So MODULE_R9M_FLEX_FW it is :)